extensions/power-management/package.json

```diff
@@ -15,22 +15,40 @@
     ],
     "license": "MIT",
     "commands": [
+        {
+            "title": "Get Power Mode Details",
+            "name": "get-low-power-detail",
+            "description": "Get detailed low power mode settings",
+            "mode": "view"
+        },
         {
             "title": "Toggle Low Power Mode",
             "name": "toggle-low-power-mode",
-            "description": "Toggle Low Power Mode of the System",
+            "description": "Toggle low power mode",
+            "mode": "no-view"
+        },
+        {
+            "title": "Enable Low Power Mode",
+            "name": "enable-low-power-mode",
+            "description": "Enable low power mode",
+            "mode": "no-view"
+        },
+        {
+            "title": "Disable Low Power Mode",
+            "name": "disable-low-power-mode",
+            "description": "Disable low power mode",
             "mode": "no-view"
         },
         {
             "name": "lowpower-menubar",
-            "title": "Menu Bar Low Power Mode",
-            "description": "See status of Low Power Mode toggle on the menubar",
+            "title": "Low Power Mode Menu",
```

Comment:
Probably should still keep the `Bar` there, eg `Low Power Mode Menu Bar`, as that's the seems to be the standard with other extensions